README.md

```diff
@@ -16,12 +16,25 @@ Or install it yourself as:
 
     $ gem install ar_state_machine
 
+## Setup
+
+config/initializers/ar_state_machine.rb
+
+```ruby
+
+  ArStateMachine.configure do |config|
+    config.system_id = [system_id]
```

Comment:
idk I wanted a way to make it be like hey put stuff here, but wasn't 100% sure if I should just do system_id